packages/spor-elm/src/Spor/LineTag/TravelTag.elm

```diff
@@ -140,7 +154,7 @@ toHtml (TravelTag options) =
                 Css.batch []
             ]
         ]
-        [ lineIcon options, lineText options ]
+        [ lineIcon options, deviationIcon options.deviationLevel, lineText options ]
```

Comment:
kunne man vurdert å sende inn kun options sånn som man gjør for lineIcon og lineText? For å få det på samme format?

Reply:
Ja, her ser jeg det har gått litt fort i svingene 👍

Reply:
Eller, tror tanken min var at for `deviationIcon` trenger jeg kun en av attributtene på options, mens for de andre trenger jeg mer enn to av dem, så derfor sendte jeg inn hele options på de to andre (siden det føltes litt overkill å sende inn hele når jeg bare trengte en attributt) :)

Her kan det jo finnes for/mot, så er åpen for tanker 👍